src/pheval/post_processing/post_processing.py

```diff
@@ -69,27 +75,51 @@ def as_dict(self):
         }
 
 
+@dataclass
+class PhEvalDiseaseResult(PhEvalResult):
+    """Minimal data required from tool-specific output for disease prioritisation."""
+
+    disease_name: str
+    disease_identifier: str
+    score: float
+
+
+@dataclass
+class RankedPhEvalDiseaseResult(PhEvalResult):
+    """PhEval disease result with corresponding rank."""
+
+    pheval_disease_result: PhEvalDiseaseResult
+    rank: int
+
+    def as_dict(self):
+        """Return PhEval disease result as dictionary."""
+        return {
+            "disease_name": self.pheval_disease_result.disease_name,
```

Comment:
Is this the only place in the entire code that knows the key names? or do you find the string "disease_name" elsewhere? 

If the latter, using strings will violates [SRP](https://en.wikipedia.org/wiki/Single-responsibility_principle)

Reply:
Yep, it is the only place that knows the key names, it's just for writing out to a .tsv file. When the file is read back in it is read as a data class so no access to dictionary key names

Reply:
Sweet thanks!